src/main/java/net/coderbot/patchwork/manifest/forge/AccessTransformerEntry.java

```diff
@@ -0,0 +1,49 @@
+package net.coderbot.patchwork.manifest.forge;
+
+import net.fabricmc.tinyremapper.MemberInstance;
+import org.objectweb.asm.commons.Remapper;
+
+public class AccessTransformerEntry {
+	private String clazzName;
+	private String memberName;
+	private boolean memberIsField = false;
+	// todo inner class support
+	public AccessTransformerEntry(String clazzName,
+			String memberName,
+			Remapper voldeToOfficial,
+			Remapper officialToIntermediary) {
+		this.clazzName = clazzName;
+		this.memberName = memberName;
+		remap(voldeToOfficial);
+		remap(officialToIntermediary);
+		System.out.println(this.memberName);
+	}
+
+	private void remap(Remapper remapper /*remapper is easier to type*/) {
```

Comment:
As I said above I suggest making this public, and then making it return `this` so that it can be chained. Then you would do `new AccessTransformerEntry("some class", "some member").remap(voldeToOfficial).remap(officialToIntermediary)`.